assets/stylesheets/forms.scss

```
@@ -172,3 +172,9 @@ select.form-control {
         background-color: $default-bg-color-darktheme;
     }
 }
+
+// fix dropdown arrow floating over text
+.custom-select-sm {
+    background-position: right 0.2rem center;
+    padding: 0 1em 0 0.5em;
+}
```

I was wondering why both - Firefox and Chromium - need such a hack. It looks like the problematic code comes from DataTable's bootstrap-specific CSS:

```
div.dataTables_wrapper div.dataTables_length select {
  width: auto;
  display: inline-block;
}
```

The `width: auto;` line causes the problem. It makes nevertheless sense to keep it because without it the box is too long. However, it would make sense to use the same selector as the other stylings and mention it in the comment:

```suggestion
// fix drop-down arrow floating over text despite `with: auto;` from DataTable's upstream CSS
div.dataTables_wrapper div.dataTables_length select {
    background-position: right 0.2rem center;
    padding: 0 1em 0 0.5em;
}
```